examples/tutorials/04_cubes_and_objects/09_custom_objects.py

```diff
@@ -0,0 +1,101 @@
+    # This will be called whenever an EvtObjectDisappeared is dispatched - so
+    # whenever an Object goes out of view.
+    if isinstance(evt.obj, CustomObject):
+        print("Cozmo stopped seeing a %s" % str(evt.obj.object_type))
+
+
+def custom_objects(robot: cozmo.robot.Robot):
+    # Add event handlers for whenever Cozmo sees a new object
+    robot.add_event_handler(cozmo.objects.EvtObjectAppeared, handle_object_appeared)
+    robot.add_event_handler(cozmo.objects.EvtObjectDisappeared, handle_object_disappeared)
+
+    # define a unique cube (44mm x 44mm x 44mm) (approximately the same size as a light cube)
+    # with a 30mm x 30mm Diamonds2 image on every face
+    cube_obj = robot.world.define_custom_cube(CustomObjectTypes.CustomType00,
+                                              CustomObjectMarkers.Diamonds2,
+                                              44,
+                                              30, 30, True)
+
+    # define a unique cube (88mm x 88mm x 88mm) (approximately 2x the size of a light cube)
+    # with a 50mm x 50mm Diamonds3 image on every face
+    big_cube_obj = robot.world.define_custom_cube(CustomObjectTypes.CustomType01,
+                                              CustomObjectMarkers.Diamonds3,
+                                              88,
+                                              50, 50, True)
+
+    # define a unique wall (150mm x 120mm (x10mm thick for all walls)
+    # with a 50mm x 30mm Circles2 image on front and back
+    wall_obj = robot.world.define_custom_wall(CustomObjectTypes.CustomType02,
+                                              CustomObjectMarkers.Circles2,
+                                              150, 120,
+                                              50, 30, True)
+
+    # define a unique box (60mm deep x 140mm width x100mm tall)
+    # with a different 30mm x 50mm image on each of the 6 faces
+    box_obj = robot.world.define_custom_box(CustomObjectTypes.CustomType03,
+                                            CustomObjectMarkers.Hexagons2,  # front
+                                            CustomObjectMarkers.Circles3,   # back
+                                            CustomObjectMarkers.Circles4,   # top
+                                            CustomObjectMarkers.Circles5,   # bottom
+                                            CustomObjectMarkers.Triangles2, # left
+                                            CustomObjectMarkers.Triangles3, # right
+                                            60, 140, 100,
+                                            30, 50, True)
+
+    if ((cube_obj is not None) and (big_cube_obj is not None) and
+            (wall_obj is not None) and (box_obj is not None)):
+        print("All objects defined successfully!")
+    else:
+        print("One or more object definitions failed!")
+        return
+
+    print("Show the above icons to Cozmo and you will see the related objects "
```

Comment:
I'll reword it to markers. You show Cozmo the marker (e.g. CustomObjectMarkers.Diamonds2) and he'll observe that object (CustomObjectTypes.CustomType00)